### DISCUSSION TITLE
How to scale sentry integration

### DISCUSSION THREAD

**Discussion by Chinwendu20** [getsentry/support#23] [Support] [answered]
Suppose I have a fleet of running services and I would like to integrate sentry into it. How do I do so to avoid repetition and also to scale the process to future services

**Comment by souredoutlook** [comment, marked as answer]
Hi @Chinwendu20,

We have a few tools for installing and managing Sentry integration at scale. For example, we support automatic instrumentation of Node and Python AWS Lambda Functions using the [AWS Lambda Integration](https://docs.sentry.io/product/integrations/cloud-monitoring/aws-lambda/).

As for managing your Sentry Organization, there is a community-supported [Terraform Integration](https://github.com/jianyuan/terraform-provider-sentry) that comes to mind, we also support SAML SCIM

Without knowing more about your services it's hard to make a concrete recommendation, I will say that Sentry SDKs have been designed to be as quick and painless to set up as possible.

**Comment by kitsunde** [comment]
1. How does that work when you add new lambda functions? Do you need to go in and enable sentry post-deployment on new functions?
2. If I set a different sentry lambda layer in my functions cloudformation does it respect that?
3. If I'm on layer 65 and deploy does it pull in a newer version or does it keep the current version?

My concern here is being able to have infra as code, and there have been some specific issues with the lambda layers breaking the runtime.
